frontend/src/pages/RunDetails.tsx

```diff
@@ -871,6 +883,19 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
         ];
   }
 
+  private _getTaskDetailsFields(workflow: Workflow, nodeId: string): Array<KeyValue<string>> {
+    return !workflow.status || !workflow.status.nodes || !workflow.status.nodes[nodeId]
```

Comment:
nit: simplify using optional chaining

Reply:
just confirmation, did you forget to push changes related to this?

Reply:
Thank you @Bobgy for pointing it out! I am not sure why this comment didn't show up on my end last time.